src/blocks/content-helper/content-helper-provider.ts

```diff
@@ -80,7 +83,8 @@ class ContentHelperProvider {
 			return Promise.reject( error );
 		}
 
-		let message = `${ __( 'Top-performing posts', 'wp-parsely' ) } ${ apiQuery.message }.`;
+		/* translators: %s: message from api, %d: default time range of related posts api */
```

Comment:
```suggestion
		/* translators: %s: message such as "from category Foo", %d: number of days */
```

Translators won't care where the placeholder values come from; they only need an example of what it could look like.